### DISCUSSION TITLE
Time series and concurrency

### DISCUSSION THREAD

**Discussion by cadilhac** [ravendb/ravendb#19502] [Q&A] [answered]
Can a client modify the content of a document and another alter its time series without any conflict?

**Comment by SbiCA** [comment]
I think you're looking for incremental timeseries 

https://ravendb.net/docs/article-page/6.2/csharp/document-extensions/timeseries/incremental-time-series/overview

there was also a pretty good webinar on that ages ago by @ayende  

- ts in general: https://youtu.be/mmY6CT4ixHc
- incremental ts: https://youtu.be/Ru9baI42jZQ?t=743

Be aware there also some limitations if you're using ETLs for instance 

Cheers

**Comment by karmeli87** [comment, marked as answer]
You can have a conflict between creating a _new_ time-series and modifying a document.
But, appending to an existing time-series and modifying a document will not generate a conflict.

**Reply by cadilhac** [reply]
I'm using an incremental time series so I think append is not possible...

**Reply by karmeli87** [reply]
Same for incremental time-series.
You can have a conflict between creating a _new_ incremental time-series and modifying the document.
But incrementing an existing one will not conflict with modifying the document.
